ocaml/typing/env.ml

```diff
@@ -2311,6 +2311,14 @@ and add_cltype ?shape id ty env =
   let shape = shape_or_leaf ty.clty_uid shape in
   store_cltype id ty shape env
 
+let add_module_lazy ~update_summary id presence mty env =
+  let md = Subst.Lazy.{mdl_type = mty;
+                       mdl_attributes = [];
+                       mdl_loc = Location.none;
+                       mdl_uid = Uid.internal_not_actually_unique}
```

Comment:
There is actually already a function `md` in scope to make dummy `module_declaration`s, so I think this can just be:
```ocaml
  add_module_declaration_lazy ~update_summary id presence (md mty) env
```

Reply:
@lpw25 I think `md` makes a `Types.module_declaration` but I need a `Subst.Lazy.module_decl`.

Reply:
Ah, my mistake. This is fine then.